bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PacketProcessorBase.java

```diff
@@ -145,12 +152,12 @@ protected void sendResponseAndWait(int rc, Object response, OpStatsLogger statsL
         try {
             ChannelFuture future = channel.writeAndFlush(response);
             if (!channel.eventLoop().inEventLoop()) {
-                future.await();
```

Comment:
what's the difference in this case ?

Reply:
`await()` won't throw the exception out. We'd better catch the exception and log it in debug mode to help debug issues.